classes/02class/exercises/c02-network03/README.md

```diff
@@ -0,0 +1,146 @@
+# AWS VPC + subnets
+
+Create a txt file under your folder name to add deliverables.
```

Comment:
would be easier to specify the name of the file, so we keep it consistent.

Reply:
don't really like the idea of specifying the name of the file but that is fine